### PR TITLE
EOS-24122: Update support.yaml format to Interim format for support.yaml file

### PR DESCRIPTION
**Pull request by satish-seagate**
Signed-off-by: Satish Darade <satish.darade@seagate.com>

## Coding:

* [ ] Did you follow coding standard and verified it with flake8? 

* [ ] Did your address all Codacy issues?

## Testing:

* [ ] Unittest updated and all unittests are passing?

* [ ] Sanity/Self tests are updated (if applicable)?

* [ ] Manual testing done covering happy path and non-happy path?

## Integration:

* [ ] If there is any interface change, did you communicate it to other components?
* [ ] If there is any interface change, other componenent gate gatekeepers are ready to accept the change?
* [ ] Did you complete deployment test if applicable?

## PR checklist:
* [ ] Did you add Jira number to the PR?

* [ ] DCO and cla-signed?

### CODE REVIEW

**Review by sumedhak27 — APPROVED** [review]

**Review by sandeepsgt — APPROVED** [review]

**Review by thavanathan — APPROVED** [review]

**Review by vastradparayya — APPROVED** [review]
+1

File: low-level/files/opt/seagate/sspl/conf/support.yaml [hunk 1]
```diff
@@ -1,45 +1,17 @@
...
-      url: /opt/seagate/cortx/sspl/bin/sspl_support_bundle create
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
 
-    status:
-      url: /opt/seagate/cortx/sspl/bin/sspl_support_bundle status
+support_bundle:
+    - /opt/seagate/cortx/sspl/bin/sspl_support_bundle create
```
Comment (cdeshmukh) [inline low-level/files/opt/seagate/sspl/conf/support.yaml:17]: what about 'status' ? create and get status supported right ?

Comment (satish-seagate) [reply low-level/files/opt/seagate/sspl/conf/support.yaml:17]: Yes, @cdeshmukh , As discussed and that was decided. but get staus is still not exposed by every component for use by support bundle framework @vastradparayya please confirm. @vastradparayya have created this ticket to deleted/modify existing support.yaml file and update with latest SB Framework confluence page. 
https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/316735509/Support+Bundle+Framework#Interim-format-for-support.yaml-file
And as I got some points from @rahul-tripathi-git . I have pasted it here for information.
> [8/26 5:35 PM] Rahul Tripathi
    Hi Satish,
Whatever we discussed holds true. As the design architecture of cortx may/will change, we have put a halt on any such (New) requirements.
You can leave the code for get_status as is for now.

> ​[8/26 11:15 PM] Satish Darade
okay.
Thanks for the confirmation.
(1 liked)

**Review by shriya-deshmukh — APPROVED** [review]

File: low-level/files/opt/seagate/sspl/conf/support.yaml (hunk 1 above)
Comment (cdeshmukh) [reply low-level/files/opt/seagate/sspl/conf/support.yaml:17]: @sachinpunadikar please confirm that create interface alone is enough right now ?

Comment (sachinpunadikar) [reply low-level/files/opt/seagate/sspl/conf/support.yaml:17]: Thats correct @cdeshmukh . Currently we need create interface only

**Review by cdeshmukh — APPROVED** [review]

**Pull request merged**